core/trino-main/src/test/java/io/trino/sql/query/QueryAssertions.java

```diff
@@ -341,6 +341,21 @@ public QueryAssert matches(MaterializedResult expected)
             });
         }
 
+        public final QueryAssert matches(PlanMatchPattern expectedPlan)
```

Comment:
let's put this in a preparatory commit

Reply:
Different PR?

Reply:
I meant 1 commit adding this method and then another commit with the rest of the changes